src/dataset-builder/DatasetBuilder.ts

```diff
@@ -704,16 +708,13 @@ export const DatasetBuilderView: React.FC<DatasetBuilderProps> = (props) => {
   };
 
   useOnMount(() => {
-    void loadDatasetModel(async () =>
-      DataRepo()
-        .dataset(datasetId)
-        .details([datasetIncludeTypes.SNAPSHOT_BUILDER_SETTINGS, datasetIncludeTypes.PROPERTIES])
-    );
+    void loadSnapshot(async () => DataRepo().snapshot(snapshotId).details());
```

Comment:
I think here we are going to run into trouble. I don't think an `AGGREGATE_DATA_READER` user has permission to see a snapshot's details, only `READER` can do that. An `AGGREGATE_DATA_READER` can only see the settings.

For now, we can provide stub data in the UI for any properties we were getting from the dataset (like name and description) but I think we'll need to add this information to the settings object in the long term.

Reply:
Ah, that's a good point. Yeah, I think that's a good approach.